genai-perf/tests/test_converters/test_dynamic_grpc_converter.py

```diff
@@ -139,7 +139,7 @@ def test_check_config_valid(valid_config):
 
 
 def test_check_config_invalid_batch_size(valid_config):
-    valid_config.input.batch_size = [2]
+    valid_config.input.batch_size = InputDefaults.BATCH_SIZE + 1
```

Comment:
How did it change from a list to an integer?

Reply:
Good catch, fixed! I don't know how this happened.